types/reactstrap/index.d.ts

```diff
@@ -43,6 +43,7 @@ export { default as CarouselCaption,          CarouselCaptionProps            }
 export { default as Col,                      ColProps                        } from './lib/Col';
 export { default as Collapse,                 CollapseProps                   } from './lib/Collapse';
 export { default as Container,                ContainerProps                  } from './lib/Container';
+export { default as CustomInput,              CustomInputProps                } from './lib/CustomInput';
```

Comment:
The version number in the header will need to be updated.

Reply:
Hello @FaithForHuman, understood, like 6.0 or do you think of another version?

Reply:
Yep, exact that. 6.0

Reply:
@FaithForHumans there you go :)